src/modules/taskCheck.js

```diff
@@ -0,0 +1,19 @@
+const list = JSON.parse(localStorage.getItem('list'));
+
+function toggleStatues(taskId) {
+  list[taskId].comp = !list[taskId].comp;
+  localStorage.setItem('list', JSON.stringify(list));
+}
+
+function getTaskId(e) {
+  return e.target.getAttribute('data-id');
+}
+
+export default function checkboxListener() {
+  document.addEventListener('click', (e) => {
+    if (e.target.type === 'checkbox') {
+      const taskId = getTaskId(e);
+      toggleStatues(taskId);
```

Comment:
- [ ] The task object's values for `completed` should be updated upon user actions.

Reply:
Can you describe which value i should change ? 
if you meant `list` variable thats defined in `index.js` 
its just an initial value for the app to run, 
i can't access it through `taskCheck.js` cuz this module is already imported by `index.js`...

@DeliceLydia, please if you have time give me more clarification

Reply:
I mean when I check the box the local storage doesn't be updated